libs/community/langchain_community/embeddings/yandex.py

```diff
@@ -76,7 +76,7 @@ class Config:
 
         allow_population_by_field_name = True
 
-    @root_validator()
```

Comment:
it causes #25575, which is fixed by #25617 
@eyurtsev could you please take a look?